samples/sensor/amg88xx/prj.conf

```diff
@@ -1,6 +1,6 @@
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_SYS_LOG_SENSOR_LEVEL=1
-CONFIG_SYS_LOG=y
+CONFIG_SENSOR_LOG_LEVEL=1
```

Comment:
should be CONFIG_SENSOR_LOG_LEVEL_INF=y,  also in other files below...

Reply:
@nashif  All comments are addressed.